scripts/publish_registry.js

```diff
@@ -25,6 +25,11 @@ dirs.forEach(name => {
   const pkg = require(`${moduleDir}/package`);
   const moduleWithVersion = `${pkg.name}@${pkg.version}`;
 
+  if (pkg.name === '@okta/okta-react-native') {
+    console.log(`Skipping ${pkg.name}`)
```

Comment:
Lets add a comment here explaining why we need to temporarily skip this package, with a GH issue link or JIRA number for reference.